src/App.js

```diff
@@ -465,8 +466,18 @@ class Routes extends Component {
 
   renderDiscover = (props) => <Discover {...props} />;
 
+  renderLanding = (props) => <Landing {...props} />;
+
   render() {
-    const { renderModalThread, renderShowPage, renderFaqPage, renderIndexPage, renderThread, renderDiscover } = this;
+    const {
+      renderModalThread,
+      renderShowPage,
+      renderFaqPage,
+      renderIndexPage,
+      renderThread,
+      renderDiscover,
+      renderLanding,
```

Comment:
To jest ta jedna linijka, zignoruj zmiane układu. 
Dodałem tylko `, renderLanding`